tests/scripts/test_cell_simulation.py

```diff
@@ -0,0 +1,62 @@
+"""Tests the cell_simulation script."""
```

Comment:
Wrote a test for the `cell_simulation.py` script here, as I spotted a bug and some errors before. As discussed this `pytest` tests a script directly. - Consider refactoring the logic from the `cell_simulation.py` script to `tree_inference.simulate`.